src/Extensibility/nunit.extensibility.tests/nunit.extensibility.tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>NUnit.Extensibility</RootNamespace>
+    <TargetFrameworks>net462;net8.0</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <OutputPath>../../../bin/$(Configuration)</OutputPath>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
+    <DebugType>Full</DebugType>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
+    <NoWarn>1505</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Please add Nullable.props and NUnit.Analyzers to suppress some errors.

Reply:
I'll do it like in nunit.core.tests. However, this approach seems to be very busy. Maybe more should be put into Directory.Build.props.

Reply:
Nullable.props can be put in Directory.build.props, bit didn't you say you liked everything specified in individual csproj files?

We could create a `nunit.props` which includes everything needed for all test assemblies.

Reply:
I'm poor at expressing myself in short bits, like here.

I'm not entirely comfortable with "surprises" when editing the csproj file, due to not seeing the result. But I see the value of centralizing some things. There's a balance point to be found, probably, and I'd like to find it.

Directory.Build.Props works well for tests when all the tests are in some directory. Currently, we recognize tests in the build script by a naming pattern rather than locations. Maybe we should chat briefly sometime and decide on a direction that would not surprise me but make life easier for all.